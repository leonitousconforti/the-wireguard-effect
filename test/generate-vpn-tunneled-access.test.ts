import { describe, it } from "@effect/vitest";

import * as Effect from "effect/Effect";

describe.skip("wireguard generate vpn tunneled access", () => {
    it.effect("Should generate configs", () => Effect.gen(function* () {}));
});
