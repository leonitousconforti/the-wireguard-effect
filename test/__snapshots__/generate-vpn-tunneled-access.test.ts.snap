// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`wireguard generate vpn tunneled access > Should generate configs 1`] = `
{
  "Address": "10.0.0.1/24",
  "ListenPort": 51820,
  "Peers": [
    {
      "AllowedIPs": Set {
        "10.0.0.2/32",
      },
      "Endpoint": undefined,
      "PresharedKey": StringMatching /\\^\\[\\\\d\\+/A-Za-z\\]\\{42\\}\\[048AEIMQUYcgkosw\\]=\\$/,
      "PublicKey": StringMatching /\\^\\[\\\\d\\+/A-Za-z\\]\\{42\\}\\[048AEIMQUYcgkosw\\]=\\$/,
    },
  ],
  "PrivateKey": StringMatching /\\^\\[\\\\d\\+/A-Za-z\\]\\{42\\}\\[048AEIMQUYcgkosw\\]=\\$/,
}
`;

exports[`wireguard generate vpn tunneled access > Should generate configs 2`] = `
{
  "Address": "10.0.0.2/24",
  "ListenPort": 0,
  "Peers": [
    {
      "AllowedIPs": Set {
        "10.0.0.1/32",
        "0.0.0.0/0",
      },
      "Endpoint": "server.wireguard.com:51820:51820",
      "PresharedKey": StringMatching /\\^\\[\\\\d\\+/A-Za-z\\]\\{42\\}\\[048AEIMQUYcgkosw\\]=\\$/,
      "PublicKey": StringMatching /\\^\\[\\\\d\\+/A-Za-z\\]\\{42\\}\\[048AEIMQUYcgkosw\\]=\\$/,
    },
  ],
  "PrivateKey": StringMatching /\\^\\[\\\\d\\+/A-Za-z\\]\\{42\\}\\[048AEIMQUYcgkosw\\]=\\$/,
}
`;
